docs/reference/data-streams/downsampling.asciidoc

```diff
@@ -0,0 +1,171 @@
+dimensions and metrics may be stored.
+
+//.Metric anatomy
+image::images/data-streams/time-series-metric-anatomy.png[align="center"]
+
+For your most current and relevant data, the metrics series typically has a low
+sampling time interval, so it's optimized for queries that require a high data
+resolution.
+
+.Original metrics series
+image::images/data-streams/time-series-original.png[align="center"]
+
+Downsampling works on older, less frequently accessed data by replacing the
+original time series with both a data stream of a higher sampling interval and
+statistical representations of that data. Where the original metrics samples may
+have been taken, for example, every ten seconds, as the data ages you may choose
+to reduce the sample granularity to hourly or daily. You may choose to reduce
+the granularity of `cold` archival data to monthly or less.
+
+.Downsampled metrics series
+image::images/data-streams/time-series-downsampled.png[align="center"]
+
+[discrete]
+[[running-downsampling]]
+=== Running downsampling on time series data
+
+To downsample a time series index, use the
+<<indices-downsample-data-stream,Downsample API>> and set `fixed_interval` to
+the level of granularity that you'd like:
+
+```
+POST /<source_index>/_downsample/<new_index>
+{
+    "fixed_interval": "1d"
+}
+```
+
+To downsample time series data as part of ILM, include a
+<<ilm-downsample,Downsample action>> in your ILM policy and set `fixed_interval`
+to the level of granularity that you'd like:
+
+```
+PUT _ilm/policy/my_policy
+{
+  "policy": {
+    "phases": {
+      "warm": {
+        "actions": {
+          "downsample" : {
+            "fixed_interval": "1h"
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+[discrete]
+[[querying-downsampled-indices]]
+=== Querying downsampled indices
+
+You can use the <<search-search,`_search`>> and <<async-search,`_async_search`>>
+endpoints to query a downsampled index. Multiple raw data and downsampled
+indices can be queried in a single request, and a single request can include
+downsampled indices at different granularities (different bucket timespan). That
+is, you can query data streams that contain downsampled indices with multiple
+downsampling intervals (for example, `15m`, `1h`, `1d`).
+
+The result of a time based histogram aggregation is in a uniform bucket size and
+each downsampled index returns data ignoring the downsampling time interval. For
+example, if you run a date_histogram aggregation with `"fixed_interval": "1m"`
+on a downsampled index that has been downsampled at an hourly resolution
+(`"fixed_interval": "1h"`), the query returns one bucket with all of the data at
+minute 0, then 59 empty buckets, and then a bucket with data again for the next
+hour.
+
+NOTE: When runnning queries in {kib} and through Elastic solutions, a normal
+response is returned without notification that some of the queried indices are
+downsampled.
+
```

Comment:
We must add the limitations when querying downsample indices:
- Only fixed_interval is supported for date_histogram aggregations
- Only UTC timezone is supported

Reply:
Added via [9c6c46c](https://github.com/elastic/elasticsearch/pull/88571/commits/9c6c46c2e597177817a1dda6519922f3e54a12c8)